src/components/clusterexplorer/node.resource.ts

```diff
@@ -56,6 +57,24 @@ export class ResourceNode extends ClusterExplorerNodeImpl implements ClusterExpl
     get isExpandable(): boolean {
         return getChildSources(this.kind).length > 0;
     }
+
+    async getPathApi(namespace: string): Promise<string> {
+        const resources = this.kind.pluralDisplayName.replace(/\s/g, '').toLowerCase();
```

Comment:
I'd strongly advise against depending on the display name here.  If none of the existing `ResourceKind` attributes are correct for the API (which I guess they might not be because `abbreviation` is closest and that may be anything `kubectl` knows how to map), I would really recommend adding a new `apiName` or `apiPath` member to `ResourceKind` and using that.  I really can't guarantee that future display names will be correct for the API and unfortunately our testing is so manual that such an issue could easily not get noticed.